desktop/main/startApp.ts

```diff
@@ -139,6 +163,10 @@ const startApp = (): AppStore => {
     // Update currentLayer & rootHash
     // Update networks on init
     fetchDiscovery($networks),
+    // trigger $runNodeBeforeLogin on openAtLoginStatus
+    makeSubscription($.of(getWasOpenAtLaunchValue()), (status) =>
```

Comment:
😱 
Why don't just `$runNodeBeforeLogin.next(getWasOpenAtLaunchValue())` after declaring the subject itself?
Or even, since we should have at least one value here on the startup, probably we can use `BehaviorSubject`, so it will become:
`const $runNodeBeforeLogin = new $.BehaviorSubject<boolean>(getWasOpenAtLaunchValue());`